### PR TITLE
blog: add release notes for 3.5.0 minor release

### PR DESCRIPTION
**Pull request by kayx23**
Released on Sept 1, 2023.

### CODE REVIEW

**Comment by kayx23** [comment]
cc: @Yilialinn @EmilyKeer @juzhiyuan

File: blog/en/blog/2023/09/01/release-apache-apisix-3.5.0.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+PR for this change is [#9607](https://github.com/apache/apisix/pull/9607).
+
+### Support the use of `allowlist` and `denylist` at the same time in `ua-restriction` plugin
+
+Support the use of `allowlist` and `denylist` at the same time in `ua-restriction` plugin.
+
+PR for this change is [#9841](https://github.com/apache/apisix/pull/9841).
+
+### Refactor and improve the plugin interface in Admin API
```
Comment (kayx23) [inline blog/en/blog/2023/09/01/release-apache-apisix-3.5.0.md:66]: @Revolyssup just an FYI. Hope this section and content look good.

File: blog/zh/blog/2023/09/01/release-apache-apisix-3.5.0.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+- 启用 L7 代理（默认启用）：`apisix.proxy_mode: http`
+- 启用 L4 代理：`apisix.proxy_mode:stream`
+- 启用 L7 和 L4 代理：`apisix.proxy_mode: http&stream`
+
+有关更改后如何使用 stream 代理的更多信息，请参阅[如何启用 stream 代理](https://apisix.apache.org/zh/docs/apisix/next/stream-proxy/#如何开启-stream-代理）。
+
+相关 PR 请见 [#9607](https://github.com/apache/apisix/pull/9607)。
+
+### 在 `ua-restriction` 插件中支持同时使用`allowlist`和`denylist`
```
Comment (AlinsRan) [inline blog/zh/blog/2023/09/01/release-apache-apisix-3.5.0.md]: This is the previous behavior.
Cannot be enabled at the same time.

File: blog/en/blog/2023/09/01/release-apache-apisix-3.5.0.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+- To enable L7 proxy (enabled by default): `apisix.proxy_mode: http`
+- To enable L4 proxy: `apisix.proxy_mode: stream`
+- To enable both L7 and L4 proxy: `apisix.proxy_mode: http&stream`
+
+For more information about how to work with stream proxy after the change, see [how to enable stream proxy](https://apisix.apache.org/docs/apisix/next/stream-proxy/#how-to-enable-stream-proxy).
+
+PR for this change is [#9607](https://github.com/apache/apisix/pull/9607).
+
+### Support the use of `allowlist` and `denylist` at the same time in `ua-restriction` plugin
```
Comment (AlinsRan) [inline blog/en/blog/2023/09/01/release-apache-apisix-3.5.0.md]: Cannot be enabled at the same time.

**Review by AlinsRan — APPROVED** [review]

**Review by AlinsRan — CHANGES_REQUESTED** [review]
sorry, wrong point.

**Review by AlinsRan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**